vulcan/hebe/data/_lesson.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+from typing import Union, AsyncIterator, List
+import datetime
+from related import (
+    immutable,
+    IntegerField,
+    StringField,
+    ChildField,
+    BooleanField,
+    SequenceField,
+    TimeField,
+)
+
+from .._api_helper import FilterType
+from .._endpoints import DATA_TIMETABLE
+from ..model import Serializable, DateTime, Teacher, Subject, TeamClass, TeamVirtual
+from ..._utils import TIME_FORMAT_H_M, sort_and_filter_dates
+
+
+@immutable
+class LessonTime(Serializable):
```

Comment:
This may be moved to `hebe.model` and renamed to `TimeSlot` for future usage in the API, e.g. in the `dictionary/timeslot` endpoint.